scripts/run_tests.psm1

```diff
@@ -76,7 +76,6 @@ function Invoke-CICDTests
     try {
 
          $TestList = @(
-            "unit_tests.exe",
```

Comment:
why this change?

Reply:
unit_tests run in other WFs. Its redundant in the driver VM runs. Besides these tests dont invoke the driver code.

Reply:
But this script is called "run_tests" not "run_vm_tests", and the current name implies it will run all tests.  If you don't mean to run all tests, then I recommend renaming this file.

Reply:
renamed file.